src/java.base/share/classes/jdk/internal/util/Architecture.java

```diff
@@ -31,85 +33,80 @@
  * The order and ordinal values must match the values defined in OperatingSystemProps.java.template.
  */
 public enum Architecture {
-    X64(true),        // Represents AMD64 and X86_64
-    X86(false),
-    IA64(true),
-    ARM(false),
-    AARCH64(true),
-    RISCV64(true),
-    S390X(true),
-    PPC64LE(true),
+    X64(),        // Represents AMD64 and X86_64
+    X86(),
+    IA64(),
+    ARM(),
+    AARCH64(),
+    RISCV64(),
+    S390X(),
+    PPC64LE(),
     ;
 
     // Cache a copy of the array for lightweight indexing
     private static final Architecture[] archValues = Architecture.values();
 
-    /**
-     * 64-bit Architecture = true; false = 32-bit/other.
-     */
-    private final boolean is64Bit;
-
-    /**
-     * Construct an Architecture enum.
-     * @param is64Bit true if the architecture uses 64-bit addressing.
-     */
-    Architecture(boolean is64Bit) {
-        this.is64Bit = is64Bit;
-    }
-
     /**
      * {@return {@code true} if the current architecture is X64, Aka amd64}
      */
+    @ForceInline
     public static boolean isX64() {
         return OperatingSystemProps.TARGET_ARCH_IS_X64;
     }
 
     /**
      * {@return {@code true} if the current architecture is X86}
      */
+    @ForceInline
     public static boolean isX86() {
         return OperatingSystemProps.TARGET_ARCH_IS_X86;
     }
 
     /**
      * {@return {@code true} if the current architecture is IA64}
      */
+    @ForceInline
     public static boolean isIA64() {
         return OperatingSystemProps.TARGET_ARCH_IS_IA64;
     }
 
     /**
      * {@return {@code true} if the current architecture is Arm}
      */
-    public static boolean isArm() {
+    @ForceInline
+    public static boolean isARM() {
         return OperatingSystemProps.TARGET_ARCH_IS_ARM;
     }
 
     /**
      * {@return {@code true} if the current architecture is RISCV64}
      */
-    public static boolean isRiscv64() {
+    @ForceInline
+    public static boolean isRISCV() {
```

Comment:
I think it should be named `isRISCV64` because RISC-V also has a 32-bit variant, so the name `isRISCV` may cause confusion.